subsystems/climbersubsystem.py

```diff
@@ -0,0 +1,89 @@
+from enum import Enum, auto
+from commands2 import Subsystem
+
+from ntcore import NetworkTableInstance
+
+from util.simtalon import Talon
+import constants
+
+
+class ClimberSubsystem(Subsystem):
+    class ClimberState(Enum):
+        TuckedPosition = auto()
+        AtFramePosition = auto()
+        EndClimbPosition = auto()
+        NothingPressed = auto()
+
+    def __init__(self) -> None:
+        Subsystem.__init__(self)
+        self.setName(__class__.__name__)
+
+        self.climberMotor = Talon(
```

Comment:
do note that the climberMotor's internal encoder will start at 0, which means that you are defining 0, since I see nowhere else where the encoder is set, as the startup position